e2e/tsconfig.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Should the ts config be empty, or should we use the same config as for the app?

Reply:
An empty tsconfig throws an error during running playwright (I think tsc does not like it). Yes, we will use similiar configs in the end, I wanted to keep this PR smallish and focus on "setup playwright with POC test running".

Reply:
I mean, `tsconfig` seems like "Initial commit" kind of thing to me :o I don't think it would overload this PR by having it defined ^^

Reply:
https://github.com/Aiven-Open/klaw/pull/1738/commits/0df9ead3865cb64526aa2e41d94580a819709491